code/modules/power/tesla/coil.dm

```diff
@@ -0,0 +1,35 @@
+/obj/machinery/power/tesla_coil
+	name = "Tesla Coil"
```

Comment:
Technically..... it should be "Tesla coil". Like French fries.
Or in code: `"/improper Tesla coil"`